setup.py

```diff
@@ -15,7 +15,7 @@
     'varint>=1.0.2,<2.0',
     'six>=1.10.0,<2.0',
     'morphys>=1.0,<2.0',
-    'base58>=1.0.2,<2.0',
+    'base58>=1.0.2,<3.0',
```

Comment:
A little late, but shouldn't this have been:
```suggestion
    'base58>=2.0,<3.0',
```